net.mkiol.SpeechNote.Addon.amd.yaml

```diff
@@ -0,0 +1,371 @@
+id: net.mkiol.SpeechNote.Addon.amd
+runtime: net.mkiol.SpeechNote
+sdk: org.kde.Sdk//5.15-22.08
+build-extension: true
+appstream-compose: false
+only-arches:
+  - x86_64
```

Comment:
this is not how you do it. flatpak-builder likely complain already because this is invalid at the top level.

Reply:
```
** (flatpak-builder:35612): WARNING **: 17:07:32.559: Unknown property only-arches for type BuilderManifest
```

I missed that :/ So, I should add `only-arches` to all modules, right? 

I want to make sure this entire expansion is built just for x86. How can I enforce this?

Reply:
Fixed in c676b9de39e4e7fdc7b761de5f770e862c817157